model/actors/miner/state.go

```diff
@@ -22,7 +22,7 @@ type MinerState struct {
 	MinerID    string `pg:",pk,notnull"`
 	OwnerID    string `pg:",notnull"`
 	WorkerID   string `pg:",notnull"`
-	PeerID     []byte
+	PeerID     string
```

Comment:
This will now need a schema migration to change column type to text

Reply:
done.